### DISCUSSION TITLE
Support Opening History as full-screen tab

### DISCUSSION THREAD

**Discussion by senposage** [Floorp-Projects/Floorp#615] [Feature Request]
the History sidebar is outmoded and hard to read on large high res displays Suggest implementing an option to open History full-screen (like chrome) with options to clear browser-data  from the history tab
 currently you can manually open the history full screen by pasting this URL into a new tab but the UX isn't designed for this and while its more readable then the sidebar this should be a dedicated feature 

chrome://browser/content/places/historySidebar.xhtml

**Comment by Telmesomething** [comment]
Don't use the sidebar. Instead hit Ctrl + Shift + H to get the "Show All History" module and then double click the title bar of that to enlarge it. 

To delete history from yesterday for example, click Yesterday in the left hand pane and then ciick one of the sites in the right hand pane. Next, hit Ctrl + A to highlight everything and then hit the Delete key. If there's one or two sites you want to keep, hold down Ctrl and click those you want to keep which removes them from the highlighted area. You can subsquently delete the rest.
